csharp/DragAndDrop.Wpf/MainWindow.xaml.cs

```diff
@@ -76,31 +76,31 @@ private void ExtractData(string name, IDataObject dataObject)
 
             StringBuilder sb =
                 new StringBuilder("=====================================================");
-            sb.AppendLine("\nEvent name:" + name);
-            sb.AppendLine("\nIDataObject:" + dataObject);
+            sb.AppendLine($"\nEvent name:{name}");
```

Comment:
It is necessary to add the space after colons in this file.
For ex.:
```suggestion
            sb.AppendLine($"\nEvent name: {name}");
```